android/source/VideoLocker/res/values-en/strings.xml

```diff
@@ -22,8 +22,6 @@
     <string name="login_title">Sign in to edX</string>
 
     <!-- Navigation Drawer Labels -->
-    <string name="name_default">Morgank</string>
-    <string name="email_default">morgan.knuston@yahoo.com</string>
```

Comment:
Can we rather change the name to <code>Name</code> and email to <code>user@domain.org</code>, just to be a placeholder?